src/ufw.yml

```diff
@@ -6,9 +6,8 @@
   roles:
     - ufw
   vars:
-    # Do not actually enforce firewall rules.  The assessors will turn
-    # UFW on if they want it.
-    state: disabled
+    # Enable ufw and enforce firewall rules by default.
+    state: enabled
```

Comment:
These lines can be removed.  `enabled` is [the default behavior](https://github.com/cisagov/ansible-role-ufw/blob/41e0944c9f2d17ce58e68812f1cc0386b6d32872/defaults/main.yml#L11-L13).

Reply:
Good call- taken care of in https://github.com/cisagov/kali-packer/pull/92/commits/04b5eb8ad1f407989606bf2e2797790d4a7c0dfb.